tools/cldr-code/src/main/java/org/unicode/cldr/util/Builder.java

```diff
@@ -15,21 +15,23 @@
 import com.ibm.icu.text.Transform;
 
 /**
- * Convenience class for building collections and maps. Allows them to be built by chaining, making it simpler to
- * set as parameters and fields. Also supplies some operations that are missing on the JDK maps and collections,
- * and provides finer control for what happens with equal elements.
- * <p>
- * You start with Builder.with(...) and end with either .get() or .freeze(). With .freeze, the result is unmodifiable
- * (but its objects may be). Examples:
+ * Convenience class for building collections and maps. Allows them to be built by chaining, making
+ * it simpler to set as parameters and fields. Also supplies some operations that are missing on the
+ * JDK maps and collections, and provides finer control for what happens with equal elements.
+ *
+ * <p>You start with Builder.with(...) and end with either .get() or .freeze(). With .freeze, the
+ * result is unmodifiable (but its objects may be). Examples:
+ *
  * <ul>
```

Comment:
This JavaDoc is invalid (bare '<' , '>' but the Google formatter formats it okay). Not worse than what's there, and the HTML structure is a bit better.